connectors/rocketmq-replicator/src/main/java/org/apache/rocketmq/replicator/ReplicatorSourceConnector.java

```diff
@@ -122,6 +135,25 @@ public int compare(MessageQueue o1, MessageQueue o2) {
         return result;
     }
 
+    private void execScheduleTask() {
+        requestTaskReconfigExecutorService.scheduleAtFixedRate(new Runnable() {
+            @Override
+            public void run() {
+                Map<String, String> topicTagMap = ReplicatorConnectorConfig.getSrcTopicTagMap(
+                        connectorConfig.getString(ReplicatorConnectorConfig.SRC_INSTANCEID),
+                        connectorConfig.getString(ReplicatorConnectorConfig.SRC_TOPICTAGS));
+                if (MapUtils.isEmpty(topicTagMap)) {
+                    throw new ConnectException("sink connector topics config cannot be null, please check sink connector config info");
+                }
```

Comment:
If an exception is thrown, the task will be interrupted and there will be no log output.

Reply:
Done.